packages/backend-test-utils/api-report.md

```diff
@@ -139,6 +144,19 @@ export interface MockDirectoryContentOptions {
   shouldReadAsText?: boolean | ((path: string, buffer: Buffer) => boolean);
 }
 
+// @public
+export function mockErrorHandler(options?: {
+  logger?: LoggerService;
+  logClientErrors?: boolean;
+  showStackTraces?: boolean;
```

Comment:
Hmm, I think we should build this specifically for tests as much as possible. I'm thinking we most likely only want the error conversion logic and none of the logging, as that's not particularly useful in tests anyway. Perhaps we just get rid of all of these options and then supply a no-op logger instead? Still best to keep it as a function though

Reply:
Makes sense, just updated it.